zebra-rpc/src/methods.rs

```diff
@@ -10,6 +10,7 @@ use std::{collections::HashSet, fmt::Debug, sync::Arc};
 
 use chrono::Utc;
 use futures::{stream::FuturesOrdered, FutureExt, StreamExt, TryFutureExt};
+use get_block_template_rpcs::types::zec::Zec;
```

Comment:
This type needs to be moved out from behind the `getblocktemplate-rpcs` feature flag.